scripts/channel_map.py

```diff
@@ -3,8 +3,8 @@
 class ChannelMap():
     channel_map = {
         'main': {
-            'tfm': 'net9.0',
-            'branch': '9.0',
+            'tfm': 'net10.0',
+            'branch': '10.0',
```

Comment:
We should do this in a separate change as this shouldn't be a dependency to get https://github.com/dotnet/runtime/pull/106599 working. This also means we can spend some time to get the BDN changes in as well to support wasm on .NET 10.

Reply:
@am11 Can you remove the change to channel_map.py then we can merge this in?